packages/indexer-common/package.json

```diff
@@ -44,6 +45,9 @@
     "lodash.groupby": "^4.6.0",
     "lodash.isequal": "4.5.0",
     "lodash.xor": "^4.5.0",
+    "lodash.transform": "^4.6.0",
+    "lodash.camelcase": "^4.3.0",
+    "lodash.snakecase": "^4.1.1",
```

Comment:
This is not used anymore.

Reply:
its used in the ravTest

Reply:
The test that uses it should be removed because we are not parsing receipts from the database anymore.